### PR TITLE
docs: added MongoDB and FaunaDB Datasources

### PR DESCRIPTION
**Pull request by Slickstef11**
#### Checklist

- [X] run `make test`
- [X] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [X] tests and/or benchmarks are included
- [X] documentation is changed or added
- [X] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs-website/src/pages/docs/supported-data-sources/mongodb-atlas.md [hunk 1]
```diff
@@ -1,5 +1,62 @@
...
+
+Simply point WunderGraph to your Database, and we take care of all the heavy lifting.
+
+In contrast to other offerings, the resulting API is production-ready from the start with authentication & authorization out of the box.
+
+No data gets exposed if you didn't explicitly allow it.
+
+## Highlights
+
+### Injecting Claims
```
Comment (StarpTech) [inline docs-website/src/pages/docs/supported-data-sources/mongodb-atlas.md:19]: This part is already covered in auth section and isn't specific to mongodb.

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: docs-website/src/pages/docs/supported-data-sources/mongodb-atlas.md (hunk 1 above)
Comment (StarpTech) [reply docs-website/src/pages/docs/supported-data-sources/mongodb-atlas.md:19]: In my opinion, this section should only inform about the compatibility and reference a full example.

Comment (Slickstef11) [reply docs-website/src/pages/docs/supported-data-sources/mongodb-atlas.md:19]: @StarpTech Do we have a full example? I know we have a PR for one. Has it been reviewed?

Comment (StarpTech) [reply docs-website/src/pages/docs/supported-data-sources/mongodb-atlas.md:19]: Not yet :cry:

File: docs-website/src/pages/docs/supported-data-sources/mongodb-atlas.md [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
+
+This method allows us to add a layer of authentication & authorization by just writing a directive.
+
+### Live Queries
+
+WunderGraph allows you to turn any GraphQL Query into a Live-Query.
+Live-Queries are regular Queries that automatically update the user interface.
+Data gets updated by polling the upstream DataSource continually using a predefined interval which you can configure.
+
+Combined with a SQLite Database, you're able to easily build Realtime Applications like a Chat.
```
Comment (jensneuse) [inline docs-website/src/pages/docs/supported-data-sources/mongodb-atlas.md:58]: if you do copy pasta, do it right :D

**Review by jensneuse — CHANGES_REQUESTED** [review]

**Pull request closed without merging**